covidadmin/urls.py

```diff
@@ -42,7 +43,7 @@
     path("api-auth/", include("rest_framework.urls")),
     path("cms/", include(wagtailadmin_urls)),
     path("documents/", include(wagtaildocs_urls)),
-    path("pages/", include(wagtail_urls)),
+    # path("pages/", include(wagtail_urls)),
     path("api/v2/", api_router.urls),
-    re_path(r"^", include(wagtail_urls)),
+    # re_path(r"^", include(wagtail_urls)),
```

Comment:
Delete these lines if not needed. Lets only add comments where we want to explain something. Lets not comment unused codes.